src/main/java/agh/edu/pl/repackaging/classes/OpenTelemetryClasses.java

```diff
@@ -77,14 +75,14 @@ public void addOpenTelemetryFolders() throws IOException {
         if ((entry.getName().startsWith("inst/") || entry.getName().startsWith("io/"))
```

Comment:
Nitpick. As you are using `entry.getName()` many times, it would be better to create a variable for that.